modules/pam_timestamp/Makefile.am

```diff
@@ -32,15 +32,15 @@ endif
 securelib_LTLIBRARIES = pam_timestamp.la
 sbin_PROGRAMS = pam_timestamp_check
 
-pam_timestamp_la_SOURCES = pam_timestamp.c hmacsha1.c sha1.c
+pam_timestamp_la_SOURCES = pam_timestamp.c hmacsha1.c sha1.c hmac_openssl_wrapper.c
```

Comment:
I do not think you want to simultaneously build and link both the wrapper and the original implementation. It should be made conditional based on whether use of OpenSSL is enabled or not. And similarly below.

Reply:
I've changed this part, please review it when possible.